omnibus/package-scripts/ddot-deb/preinst

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+#
+# Perform necessary datadog-agent-ddot setup steps before package is installed.
+#
+# .deb: STEP 2 of 5
+
+SERVICE_NAME=datadog-agent-ddot
+
+set -e
+
+if [ -f "/lib/systemd/system/$SERVICE_NAME.service" ] || [ -f "/usr/lib/systemd/system/$SERVICE_NAME.service" ]; then
+    # Stop an already running agent
+    # supports systemd, upstart and sysvinit
+    if command -v systemctl >/dev/null 2>&1; then
+        systemctl stop $SERVICE_NAME || true
+    elif command -v initctl >/dev/null 2>&1; then
+        initctl stop $SERVICE_NAME || true
+    elif command -v service >/dev/null 2>&1; then
+        service $SERVICE_NAME stop || true
+    else
+        echo "[ WARNING ]\tCannot detect a supported init system. The datadog-agent-ddot package only provides service files for systemd, upstart and sysvinit."
```

Comment:
Without `-e`, a literal `\t` would be printed. Better use `printf` anyway, and write unexpected cases to the standard error.
```suggestion
        printf "[ WARNING ]\tCannot detect a supported init system. The datadog-agent-ddot package only provides service files for systemd, upstart and sysvinit." >&2
```

Reply:
Sounds good, I can make these changes in my PR. These lines are also mirrored from the files in `agent-deb` and `agent-rpm` in case you want to edit those as well